server/src/controllers/restaurant.ts

```diff
@@ -0,0 +1,28 @@
+import { Request, Response } from "express";
+import RestaurantService from "../services/restaurant";
+
+async function findNearbyRestaurants(
+  request: Request,
+  response: Response
+): Promise<void> {
+  try {
+    const latitude: number = parseFloat(request.query.latitude as string);
+    const longitude: number = parseFloat(request.query.longitude as string);
+    const meters: number = parseInt(request.query.meters as string);
```

Comment:
I like this rename form `search_radius` from `meters`. Makes a bit more sense. But lmk if you wanna do it this way, bc I might have some tests that use specificly `request.query.search_radius`